ecs-init/config/common.go

```diff
@@ -32,12 +32,13 @@ const (
 
 	UnixSocketPrefix = "unix://"
 
-	// Default region name
+	//DefaultRegionName is the name of the region to fall back to if no entry for the region name is found in the
```

Comment:
[multiple occurrences] Comments should have a space between `//` and the description: https://golang.org/doc/effective_go.html#commentary

`// DefaultRegionName`